store/event.go

```diff
@@ -1,91 +1,93 @@
 package store
+
 import (
-    "context"
-    "fmt"
-    "time"
-    "github.com/jmoiron/sqlx"
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/jmoiron/sqlx"
 )
 
 type Event struct {
-    ID                  int       `db:"id" json:"id"`
-    Name                string    `db:"name" json:"name"`
-    Description         string    `db:"description" json:"description"`
-    StartDate           time.Time `db:"start_date" json:"startDate"`
-    EndDate             time.Time `db:"end_date" json:"endDate"`
-    VolunteersRequired  int       `db:"volunteers_required" json:"volunteersRequired"`
-    CreatedAt           time.Time `db:"created_at" json:"createdAt"`
+	ID                 int       `db:"id" json:"id"`
+	Name               string    `db:"name" json:"name"`
+	Description        string    `db:"description" json:"description"`
+	StartDate          time.Time `db:"start_date" json:"startDate"`
+	EndDate            time.Time `db:"end_date" json:"endDate"`
+	VolunteersRequired int       `db:"volunteers_required" json:"volunteersRequired"`
+	Location           string    `db:"location" json:"location"`
+	CreatedAt          time.Time `db:"created_at" json:"createdAt"`
 }
 
 type EventStore interface {
-    GetAll(ctx context.Context) ([]Event, error)
-    GetOne(ctx context.Context, id int) (*Event, error)
-    GetOneByStartDate(ctx context.Context, startDate time.Time) (*Event, error)
-    GetOneByName(ctx context.Context, name string) (*Event, error) // ??
-    Create(ctx context.Context, event Event) (*Event, error)
-    Update(ctx context.Context, event Event) (*Event, error)
-    Delete(ctx context.Context, id int) error
+	GetAll(ctx context.Context) ([]Event, error)
```

Comment:
I don't know how this happened, but this isn't in line with gofmt format.

Reply:
Could you please provide a reference for gofmt format?

Reply:
vscode should have gofmt plugin enabled by default.  Let me know if you have issue with enabling that, I am more than happy to format the file and push changes. But as a side note I probably should add format check to our CI pipeline.